src/Core/Ast/Parser/Cache/CacheableFileSubscriber.php

```diff
@@ -10,7 +10,8 @@
 
 class CacheableFileSubscriber implements EventSubscriberInterface
 {
-    public function __construct(private readonly AstFileReferenceDeferredCacheInterface $deferredCache)
+    public function __construct(
+        private readonly AstFileReferenceDeferredCacheInterface $deferredCache)
```

Comment:
This looks off to me. Maybe we should fine tune the php-cs-fixer for this

Reply:
#1195